contrib/guix/libexec/build.sh

```diff
@@ -196,19 +260,60 @@ export PATH="${BASEPREFIX}/${HOST}/native/bin:${PATH}"
             find "${DISTNAME}/lib" -type f -print0
         } | xargs -0 -n1 -P"$MAX_JOBS" -I{} "${DISTSRC}/contrib/devtools/split-debug.sh" {} {} {}.dbg
 
-        cp "${DISTSRC}/doc/README.md" "${DISTNAME}/"
+        case "$HOST" in
+            *mingw*)
+                cp "${DISTSRC}/doc/README_windows.txt" "${DISTNAME}/readme.txt"
+                ;;
+            *linux*)
+                cp "${DISTSRC}/doc/README.md" "${DISTNAME}/"
```

Comment:
My build currently fails on this step with `cp: cannot stat '/bitcoin/distsrc-x86_64-linux-gnu/doc/README.md': No such file or directory`. I don't think the README is actually moved to the doc directory, or am I missing something? At least I cannot see a step for it to be copied in the top level `Makefile.am`, unlike for `README_windows.txt` which is listed.

Reply:
True! You're right. I think this is a result of #18331, which I'm seeing if I can fix with #18478

Reply:
Why can't this copy from the source dir and not from the dist dir. E.g. `cp /bitcoin/doc/README.md ...`?